tests/test_flows.py

```diff
@@ -280,6 +289,18 @@ def run_tests(self, tests):
 
     # TEST DEFINITIONS #
 
+    def test_dev_reset_default(self):
+        '''Check behaviour of dev_reset_default CAPI command.'''
+        agent = env.agents[0]
+        agent.cmd_reply("dev_reset_default,devrole,agent,program,map,type,DUT")
+        env.checkpoint()
+        time.sleep(2)
+        self.check_no_cmdu_type("autoconfig search while in reset", 0x0007, agent.mac)
+        env.checkpoint()
+        agent.cmd_reply("dev_set_config,backhaul,eth")
+        time.sleep(1)
+        self.check_cmdu_type("autoconfig search", 0x0007, agent.mac)
```

Comment:
Shouldn't you also test
```
agent.cmd_reply("dev_set_config,backhaul,radio")
```
at this point? I though that was the point of the PR ...

Reply:
There's a separate commit that adds that test. I wanted to have a baseline first. I could have done that in a separate PR.

BTW, the syntax is dev_set_config,backhaul,<radio_uid>.

(I suspect that you reviewed this while still in draft).